src/blocks/Image/ComponentClient.tsx

```
@@ -0,0 +1,39 @@
+// organize-imports-ignore
+import React from 'react'
+import { usePayloadAPI } from '@payloadcms/ui'
+import { Img } from '@react-email/components'
+import type { BlockRendererClientProps } from '../../types.js'
+
+export const ImageBlockClient = (props: BlockRendererClientProps) => {
+  const { block, imageCollectionSlug } = props
+  if (block.blockType !== 'image') {
+    return null
+  }
+  const { image, width, height, objectFit, style } = block
```

ESLint

[error] Expected "height" to come before "width".

(perfectionist/sort-objects)